### ISSUE TITLE
cgpeers does not work

### ISSUE THREAD

**Issue by puthre** [Prowlarr/Prowlarr#2347]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

Addig cgpeers shows the error: "
Unable to connect to indexer. Unexpected response status Redirect code from indexer request: Check the logs for more information."

The logs show

2025-02-14 09:00:57.7|Warn|CGPeers|Unable to connect to indexer

[v1.30.2.4939] NzbDrone.Core.Indexers.Exceptions.IndexerException: Unexpected response status Redirect code from indexer request: Check the logs for more information.
   at NzbDrone.Core.Indexers.Definitions.Gazelle.GazelleParser.ParseResponse(IndexerResponse indexerResponse) in ./Prowlarr.Core/Indexers/Definitions/Gazelle/GazelleParser.cs:line 38
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchPage(IndexerRequest request, IParseIndexerResponse parser) in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 606
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 852
RequestUri: https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc;StatusCode: Redirect;ContentType: text/html; charset=utf-8;ContentLength: 0

2025-02-14 09:00:57.7|Warn|ProwlarrErrorPipeline|Invalid request Validation failed:
 -- : Unable to connect to indexer. Unexpected response status Redirect code from indexer request: Check the logs for more information.

### Expected Behavior

Should be able to connect to cgpeers

### Environment

```markdown
- OS: Ubuntu
- Prowlarr: 1.30.2.4939
- Docker Install: yes
- Using Reverse Proxy: no
- Browser: Firefox
```

### What branch are you running?

Master

### Trace Logs?

2025-02-14 09:14:05.3|Trace|Http|Req: 192899 [POST] /api/v1/indexer/test?forceTest=true (from 192.168.1.246 Mozilla/5.0 (X11; Linux x86_64; rv:134.0) Gecko/20100101 Firefox/134.0)
2025-02-14 09:14:05.3|Debug|Prowlarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2025-02-14 09:14:05.4|Debug|CGPeers|Downloading Feed https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc
2025-02-14 09:14:05.4|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2025-02-14 09:14:05.4|Trace|IndexerHttpClient|Req: [GET] https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc
2025-02-14 09:14:05.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2025-02-14 09:14:06.1|Trace|IndexerHttpClient|Res: HTTP/2.0 [GET] https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc: 302.Redirect (0 bytes) (712 ms)
2025-02-14 09:14:06.1|Trace|CGPeers|Attempting to re-auth based on indexer search response
2025-02-14 09:14:06.1|Trace|RateLimitService|Rate Limit triggered, delaying 'cgpeers.to' for 1.234 sec
2025-02-14 09:14:07.4|Trace|IndexerHttpClient|Req: [POST] https://cgpeers.to/login.php: username=(removed)&password=(removed)&keeplogged=1
2025-02-14 09:14:07.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2025-02-14 09:14:07.7|Trace|IndexerHttpClient|Res: HTTP/2.0 [POST] https://cgpeers.to/login.php: 302.Redirect (0 bytes) (350 ms)
2025-02-14 09:14:07.7|Trace|IndexerHttpClient|Response content (0 bytes):
2025-02-14 09:14:07.7|Trace|EventAggregator|Publishing IndexerAuthEvent
2025-02-14 09:14:07.7|Trace|EventAggregator|IndexerAuthEvent -> HistoryService
2025-02-14 09:14:07.8|Trace|EventAggregator|IndexerAuthEvent <- HistoryService
2025-02-14 09:14:07.8|Debug|CGPeers|Gazelle authentication succeeded.
2025-02-14 09:14:07.8|Trace|RateLimitService|Rate Limit triggered, delaying 'cgpeers.to' for 1.559 sec
2025-02-14 09:14:09.4|Trace|IndexerHttpClient|Req: [GET] https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc
2025-02-14 09:14:09.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2025-02-14 09:14:09.5|Trace|IndexerHttpClient|Res: HTTP/2.0 [GET] https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc: 302.Redirect (0 bytes) (126 ms)
2025-02-14 09:14:09.5|Trace|CGPeers|Unexpected Response content (0 bytes):
2025-02-14 09:14:09.5|Warn|CGPeers|Unable to connect to indexer

[v1.30.2.4939] NzbDrone.Core.Indexers.Exceptions.IndexerException: Unexpected response status Redirect code from indexer request: Check the logs for more information.
   at NzbDrone.Core.Indexers.Definitions.Gazelle.GazelleParser.ParseResponse(IndexerResponse indexerResponse) in ./Prowlarr.Core/Indexers/Definitions/Gazelle/GazelleParser.cs:line 38
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchPage(IndexerRequest request, IParseIndexerResponse parser) in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 606
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in ./Prowlarr.Core/Indexers/HttpIndexerBase.cs:line 852
RequestUri: https://cgpeers.to/ajax.php?action=browse&order_by=time&order_way=desc;StatusCode: Redirect;ContentType: text/html; charset=utf-8;ContentLength: 0

2025-02-14 09:14:09.5|Trace|ProwlarrErrorPipeline|Handling Exception
2025-02-14 09:14:09.5|Warn|ProwlarrErrorPipeline|Invalid request Validation failed:
 -- : Unable to connect to indexer. Unexpected response status Redirect code from indexer request: Check the logs for more information.
2025-02-14 09:14:09.5|Trace|Http|Res: 192899 [POST] /api/v1/indexer/test?forceTest=true: 400.BadRequest (4220 ms)
2025-02-14 09:14:09.5|Debug|Api|[POST] /api/v1/indexer/test?forceTest=true: 400.BadRequest (4220 ms)
2025-02-14 09:14:10.7|Trace|Scheduler|Pending Tasks: 0

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [x] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by H4Z4RD-H42** [comment]
> There is now a TOTP option in cgpeers.

Updated now to 1.36.3.5071, but not see options to enable or set TOTP in cgpeers.

**Comment by solitonmedic** [comment]
Is there a fix for this?

**Comment by garfield69** [comment]
CGPeers have migrated from gazelle to luminance.
On Jackett yesterday we removed the C# gazelle CGPeers indexer and added a yaml luminance CGPeers indexer.
But I see that bakerboy448 has opted to block the yaml indexer from importing to Prowlarr, so presumably the Prowlarr team are taking a different upgrade path with this indexer.

**Comment by bakerboy448** [comment]
We'll migrate it over to yml shortly. I just wanted to avoid two of the same definition until I had time to deprecate the c# version - at least in nightly

**Comment by bakerboy448** [comment]
- https://github.com/Prowlarr/Indexers/commit/9c00623f5c3377ad60e47f397f42402943e4b92e
- https://github.com/Prowlarr/Indexers/commit/cffc87fe8399697eb9ed3068207685697b83c06a
- https://github.com/Prowlarr/Prowlarr/commit/f5f98e4f53a8045dc2a24e21a62e29de5bd9db17
